pkg/gpu/cgroups.go

```diff
@@ -0,0 +1,119 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build linux
+
+package gpu
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/security/utils"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// ConfigureDeviceCgroupsForProcess configures the cgroups for a process to allow access to the NVIDIA character devices
+func ConfigureDeviceCgroupsForProcess(pid uint32, rootfs string) error {
```

Comment:
rename to `ConfigureDeviceCgroups`, the comment above + pid as an input parameter are enough to mention it is done for a process